autosklearn/__init__.py

```diff
@@ -10,7 +10,7 @@
 requirements = pkg_resources.resource_string('autosklearn', 'requirements.txt')
 requirements = requirements.decode('utf-8')
 
-dependencies.verify_packages(requirements)
+# dependencies.verify_packages(requirements)
```

Comment:
Do you want to leave this line commented out?

Reply:
No, the issue is that the dependency parser cannot cope with the `git+https://github.com/automl/smac3@development`. Therefore it's temporarily commented out.